README.md

```diff
@@ -51,7 +51,7 @@ When using the "MQTT" feature, connecting to a MQTT broker using raw ip doesn't
 
 ## API reference
 
-You can read the [API reference](https://docs.rs/iota-core) here, or generate it yourself.
+You can read the [API reference](https://docs.rs/iota-lib-rs/) here, or generate it yourself.
```

Comment:
You can use "https://client-lib.docs.iota.org/docs/doc/iota/index.html" and fix #497 with that :)